snuba/admin/clickhouse/predefined_querylog_queries.py

```diff
@@ -112,3 +112,57 @@ class QueryDurationForReferrerByProject(QuerylogQuery):
     )
     ORDER BY projects ASC, time ASC
     """
+
+
+class BeforeAfterBytesScannedComparison(QuerylogQuery):
+    """Given a certain time that abuse started on a certain cluster, specify a time range before and after the abuse (recommend 30 minutes). This will show the referrers with the largest change in bytes_scanned. In the example, the abuse started on Dec 7. at 21:54:50."""
+
+    sql = """
+    SELECT referrer, before_scanned, after_scanned, (after_scanned - ifNull(before_scanned, 0)) as diff, (diff / if(equals(after_scanned, 0), 1, after_scanned))  * 100 as pct_diff
+    FROM
+    (
+        SELECT referrer, sum(arrayReduce('sum', clickhouse_queries.bytes_scanned)) as after_scanned
+        FROM querylog_local
+        WHERE timestamp >= toDateTime('2022-12-07T21:55:00')
+        AND timestamp <= toDateTime('2022-12-07T22:20:00')
```

Comment:
```suggestion
        WHERE timestamp >= toDateTime(<time_str_after_abuse>)
        AND timestamp <= toDateTime(<time_str_after_abuse + 30 mins)
```

Reply:
Making these explicit descriptive parameters may be easier for the user

Reply:
I filed a [ticket](https://getsentry.atlassian.net/browse/SNS-1931) to make this a feature in the tool, but I do think this is still an improvement

Reply:
I deliberately left these like this, since I find it easier when I have an actual example to edit. Otherwise I have to remind myself of the exact date format Clickhouse expects.

I would like to have our tools ultimately work using parameters though, in a similar way to how saved queries work on Redash. Users fill out a form with the values they want and we template them into the query.